src/coreclr/jit/emitwasm.cpp

```diff
@@ -149,22 +156,37 @@ unsigned emitter::instrDesc::idCodeSize() const
     assert(FitsIn<uint8_t>(GetInsOpcode(idIns())));
     switch (idInsFmt())
     {
+        case IF_NONE:
+            return 0;
```

Comment:
```suggestion
```
We should never produce this kind of instrDesc.

Reply:
They appear in emitxarch and I was hitting them while testing, will test removing it